docker-compose/docker-compose.yml

```
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+  postgres:
+    image: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: p0stgr3s_pass
+    ports: 
+      - '5432:5432'
+    volumes:
+      - calendso-db:/var/lib/postgresql/data
+
+  calendso:
+    image: calendso
+    environment:
+      DATABASE_URL: 'postgresql://postgres:p0stgr3s_pass@172.17.0.1:5432'
```

IPs can change depending on the local configuration. It's recommended to use the service name. It's also important to specify the name of the database

```suggestion
      DATABASE_URL: 'postgresql://postgres:p0stgr3s_pass@postgres:5432/calendso'
```